rust/src/types/opcode.rs

```diff
@@ -148,7 +148,7 @@ const REVERT: u8 = 0xFD;
 const INVALID: u8 = 0xFE;
 const SELFDESTRUCT: u8 = 0xFF;
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
I don't have any problem with making opcode copy.  passing it by value should never be a problem